### ISSUE TITLE
Coldness

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6785]
**Preferred term label:**
Coldness

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I think this is probably best capture by the existing HPO term "Chills"
